data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/PipelineTransformationConfiguration.java

```diff
@@ -10,30 +10,34 @@
 import org.springframework.context.annotation.Configuration;
 
 import javax.inject.Named;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 @Configuration
 public class PipelineTransformationConfiguration {
     public static final String TEMPLATES_DIRECTORY_PATH = "TEMPLATES_DIRECTORY_PATH";
-    public static final String RULES_DIRECTORY_PATH = "VALIDATORS_DIRECTORY_PATH";
+    public static final String RULES_DIRECTORY_PATH = "RULES_DIRECTORY_PATH";
+    private static final Path currentDir = Paths.get(System.getProperty("user.dir"));
```

Comment:
`user.dir` is the path where java was run. May not work for all cases.

Reply:
Use `data-prepper.dir` to get the runtime directory.

https://github.com/opensearch-project/data-prepper/blob/4f3ab967a14ba23d3957ad6295da0467282cf029/data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/geoip/extension/MaxMindConfig.java#L51